docs/source/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -1900,6 +1900,86 @@ These are not currently modeled.
 #. Currently mailing addresses are the same as home addresses. This is not true, especially for rural populations. We plan to add this to the model later. 
 
 
+Social Security Observer
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+Social security information is kept and tracked for a range 
+of different actions. For simplicity, we will limit this 
+section to only creation and dates of death. Others could be 
+added later, if desired. 
+
+**When to Sample** 
+
+- A sample of qualifying events is taken on every time step 
+
+**What to Sample** 
+
+.. list-table:: Simulant Attributes to Sample 
+  :widths: 20
+  :header-rows: 0
+
+  * - Unique simulant ID (for PRL tracking)
+  * - First name
+  * - Middle initial 
+  * - Last name
+  * - DOB 
+  * - Social Security Number 
+  * - Type of event 
+  * - Date of event 
+
+Currently, we will only track 2 types of events: 
+
+#. Creation of a SSN 
+#. Date of death recorded 
+
+The creation of a SSN can be triggered by a birth or by immigration into 
+the US. Both will be listed as "Creation" and are not differentiated. This 
+event is only recorded for simulates that receive a SSN. 
+The date of event is either the date of birth or the date of immigration. 
+If these values are unknown, any date within the time step can be recorded. 
```

Comment:
Fine to have this here, but my understanding is that these dates will always be known.

Reply:
Okay! I wasn't sure if simulants got an immigration "date" or just that they moved on a time step. If they get a date then I can delete this.